### ISSUE TITLE
Curios Compatibility/QoL

### ISSUE THREAD

**Issue by HeavyDeception** [henkelmax/gravestone#119]
**Is your feature request related to a problem? Please describe.**
When you pick up a grave and have [Curios API](https://www.curseforge.com/minecraft/mc-mods/curios) Forge installed the times will spill on the ground or be placed in random spots in your inventory.

**Describe the solution you'd like**
If the items could be played in the slots they came out of.

**Describe alternatives you've considered**
If your inventory is full could they be put in a pouch that goes away when you remove all items from it?

**Additional context**
It's just a Quality of Life thing'

**Comment by satoon10** [comment]
Kinda seems like this is never going to be added.

**Comment by MarioSMB** [comment]
`keepCurios = "ON"` in curios-server.json  
The developer has shown no interest in supporting modded inventory slots.

**Comment by Proxwian** [comment]
That's ridiculous

**Comment by satoon10** [comment]
> That's ridiculous

it is

**Comment by ZsoltiHUB** [comment]
Crazy

**Comment by Octelly** [comment]
epic issue comments for someone who's subscribed to this thread :+1:

**Comment by MarioSMB** [comment]
Yeah - let's not spam the tracker with comments like this. It's unfortunate that the developer has taken a stance that makes the mod annoying to use with any modern modpack, but this won't encourage them to fix it.  
The aforementioned workaround will at least make the situation manageable.
